ui/app/AppLayouts/Communities/panels/MembersTabPanel.qml

```diff
@@ -205,7 +204,7 @@ Item {
         ProfileContextMenu {
             id: memberContextMenuView
             store: root.rootStore
-            myPublicKey: userProfile.pubKey
+            myPublicKey: Global.userProfile.pubKey
```

Comment:
Ideally it should be passed from outside to limit dependencies on props from statefull singletons

Reply:
I agree but this at least is something we can mock ;)